Core/src/main/java/com/plotsquared/core/plot/world/SinglePlotManager.java

```diff
@@ -57,11 +57,11 @@ public SinglePlotManager(@Nonnull final PlotArea plotArea) {
     }
 
     @Override public Location getPlotBottomLocAbs(@Nonnull final PlotId plotId) {
-        return Location.at(plotId.toCommaSeparatedString(), -30000000, 0, -30000000);
+        return Location.at(plotId.toSeparatedString("_"), -30000000, 0, -30000000);
     }
 
     @Override public Location getPlotTopLocAbs(@Nonnull final PlotId plotId) {
-        return Location.at(plotId.toCommaSeparatedString(), 30000000, 0, 30000000);
+        return Location.at(plotId.toSeparatedString("_"), 30000000, 0, 30000000);
```

Comment:
Add a `toUnderscoreSeparatedString()` to `PlotId`, as the behaviour is repeated so often. Also consider making `30000000` a constant (`MAX_COORDINATE` or whatever)